msvc/libmonoruntime-common.targets.filters

```diff
@@ -175,7 +175,7 @@
     <ClInclude Include="$(MonoSourceLocation)\mono\metadata\loader-internals.h">
       <Filter>Header Files$(MonoRuntimeFilterSubFolder)\common</Filter>
     </ClInclude>
-    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\locales.c">
+    <ClCompile Include="$(MonoSourceLocation)\mono\culture\locales.c">
       <Filter>Source Files$(MonoRuntimeFilterSubFolder)\common</Filter>
     </ClCompile>
     <ClInclude Include="$(MonoSourceLocation)\mono\metadata\locales.h">
```

Comment:
this also seems like it should be culture

Reply:
@kg I tried (https://github.com/mono/mono/pull/19912#issuecomment-639061212):
> Did not move `locales.h` because a surprising number of files depend on it and the PR got noisy. We can do some cleanup later.

Reply:
makes sense, given that i didn't see any mistakes